Ports/zstd/package.sh

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env -S bash ../.port_include.sh
 port=zstd
+description="Zstandard"
 version=1.5.2
+website="https://facebook.github.io/zstd/"
```

Comment:
Please use single quotes (`'`) for uninterpreted strings in Bash

Reply:
sure thing!